kubernetes/utils/create_from_yaml.py

```diff
@@ -60,46 +66,69 @@ def create_from_yaml(
         processing of the request.
         Valid values are: - All: all dry run stages will be processed
     """
+    if path.exists(yaml_file):
+        with open(path.abspath(yaml_file)) as f:
+            content = f.read()
+            try:
+                yaml_file = StringIO(content)
+            except TypeError:
+                yaml_file = StringIO(content.decode('utf-8'))
+
+    yml_document_all = yaml.safe_load_all(yaml_file)
+
+    for yml_document in yml_document_all:
+        create_from_dict(k8s_client, yml_document, verbose,
```

Comment:
not resolved,
https://github.com/kubernetes-client/python/pull/795/files#r295928529

Reply:
fixed in f30f107b4258aca3a87968881d3486ba668c4776